index.html

```diff
@@ -0,0 +1,231 @@
+
+    </section>
+
+    <section id="sec-availability">
+      <h3>Availability</h3> 
+
+      <p>The <a>Public CD</a> shall be made available to the public in a form approved by the Director of Standards Development and shall be accompanied by the notices specified in <a href="#sec-availability-notices"></a>.</p>
+
+    </section>
+
+    <section id="sec-comment-period">
+      <h3>Comment Period</h3> 
+
+      <p>A Project Group should, but is under no obligation to, address comments received from the public on a <a>Public CD</a>.</p>
+      <p>The review period of a <a>Public CD</a> starts upon availability, as defined in <a href="#sec-availability"></a>, and ends when either:</p>
+      <ul>
+        <li>the document begins its FCD ballot (<a href="#bib-smpte-standards-operation-manual-v31"></a>, Section 6.6), at which point the notice specified in <a href="#sec-notice-of-fcd"></a> shall be displayed prominently alongside the <a>Public CD</a>; or</li>
+        <li>the <a>Public CD</a> is terminated.</li>
+      </ul>
+      
+
+    </section>
+
+    <section id="sec-revision">
+      <h3>Revision</h3> 
+
+      <p>A <a>Public CD</a> may be revised at any time.</p>
+      <p>To revise a <a>Public CD</a>, the Committee Draft shall be revised.</p>
+      <p>Revising a Committee Draft returns the document to Working Draft status and the process at <a href="#bib-smpte-standards-operation-manual-v31"></a>, Section 6.5 is followed to elevate the revised Working Draft to Committee Draft.</p>
+      <p>The revised Committee Draft may follow the process of <a href="#sec-creation"></a> to be made available as a <a>Public CD</a>.</p>
+
+    </section>
+
+    <section id="sec-termination">
+      <h3>Termination</h3> 
+
+      <p>A <a>Public CD</a> shall be terminated if any one of the following criteria is satisfied:</p>
+      <ul>
+        <li>the engineering document is published (see <a href="#bib-smpte-standards-operation-manual-v31"></a>, Section 6.11)</li>
+        <li>the Committee Draft of the document is revised</li>
+        <li>more than 1095 days have elapsed since the document was initially made available as a <a>Public CD</a>, unless the Standards Vice President elects to extend the period upon recommendation of the TC</li>
+        <li>the Project authorizing the engineering document is terminated</li>
+      </ul>
+      <p>Upon termination of a <a>Public CD</a>, its public access to shall be immediately terminated.</p>
+
+    </section>
+
+  </section>
+
+  <section class="annex" id="sec-process-flowchart">
+    <h2>Process Flowchart (Informative)</h2>
+
+    <figure id="figure-process-flowchart">
+      <figcaption>Example of the <a>Public CD</a> Process</figcaption>
+      <img class="figure" src="figure-process-flowchart.svg"></img>
+      <p class="note">A second Public CD (PCD2) is made available after addressing comments received on the first Public CD (PCD1).</p>
```

Comment:
This is supposed to be part of the figure caption.

Reply:
I updated it slightly for better reading in https://github.com/SMPTE/ag-22/pull/3/commits/497b350fccb5b6a443d0d0c3f0fc443c77e1d6f5

Reply:
this is now resolved with new tooling in https://github.com/SMPTE/ag-22/commit/52e2765d576711f386f7260bf99309041d9976dd